react/features/base/settings/functions.js

```diff
@@ -70,6 +74,15 @@ export function getPropertyValue(
         }
     }
 
+    // settings
+    if (sources.settings) {
+        const value = state['features/base/settings'][propertyName];
+
+        if (typeof value !== 'undefined') {
+            return value;
+        }
+    }
+
```

Comment:
The more I let the change-set soak in the more I'm being drawn to there being one settings feature instead of settings and profile. While "profile" feels a bit more user representation focused to me over settings, it makes sense from the actual features presented in the app (and soon in the web app) that they be nested into one. I believe mobile already has profile and settings in one panel and some day soon mobile will have profile and settings pushed into one pane. I don't know but I feel it might simplify the code a little (or just make it worse but being a dump hole).